Tools/dea_tools/landcover.py

```diff
@@ -273,12 +271,20 @@ def get_layer_name(measurement, da):
 
 def make_colorbar(fig, ax, measurement, horizontal=False, animation=False):
     """
-    Adds a new colorbar with appropriate land cover colours and labels
+    Adds a new colorbar with appropriate land cover colours and labels.
     
-    measurement = land cover measurement to use for colour map and lables
+    For level 4, must be used with a double plot, `ax` should be the left side canvas
+    the colour bar will go over the right hand side.
```

Comment:
This sentence is hard to read for me and I'm not sure what exactly it means. Can I suggest splitting it up as something like the following:

"For level 4 data(?), this function must be used with a double plot. The 'ax' should be on the left side of the canvas (can we use figure instead of canvas?) and the colour bar will added on the right hand side."

Reply:
Yep, that sounds much better. Updated!